tests/configuration.py

```diff
@@ -0,0 +1,193 @@
+
+
```

Comment:
I worry about user credentials etc to be visible to the rest of the community.

Reply:
we can write a dummy credentials for now, and keep that configuration file in confluence page somewhere